recreate.go

```diff
@@ -0,0 +1,437 @@
+// +build !cassandra
+// Copyright (C) 2017 ScyllaDB
+
+package gocql
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"sort"
+	"strings"
+	"text/template"
+)
+
+func (km *KeyspaceMetadata) Recreate() (string, error) {
```

Comment:
Worth a doc.

Why is this called recreate? Something like `ToCQL` would make more sense to me, but if that's a common naming in other drivers let it be.